demo/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link href="../atvImg.css" rel="stylesheet" type="text/css">
+    <script src="../atvImg.js"></script>
+    <style>
+        body {
+            display: flex;
+            justify-content: space-around;
+            align-items: center;
+            height: 400px;
+        }
+        .atvImg {
+            width: 320px;
+            height: 190px;
+        }
+    </style>
+</head>
+<body>
+    <div class="atvImg">
+        <img src="./images/kloc-icon-flattened.jpg">
+        <div class="atvImg-layer" data-img="./images/back.png"></div>
+        <div class="atvImg-layer" data-img="./images/front.png"></div>
+    </div>
+    <script>
+        atvImg();
+    </script>
+</body>
+</html>
```

Comment:
It has a “no newline at end of file” warning. Those are [actually meaningful](http://unix.stackexchange.com/a/18789).